### PR TITLE
Add 手机百度 to the showcase page

### PR DESCRIPTION
**Pull request by zimengle**

### CODE REVIEW

**Comment by facebook-github-bot-2** [comment]
By analyzing the blame information on this pull request, we identified @lacker and @alexleventer to be potential reviewers.

**Comment by facebook-github-bot-2** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-0** [comment]
@zimengle updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9845/files/acbc853e9446e6eaf744634acca627d313e356a1..5829fc75d7710a31b4b6495423e6323a0448cb40)

**Comment by facebook-github-bot-3** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by nihgwu** [comment]
I think you can check this PR #9807 to refine yours for sooner merge

**Comment by facebook-github-bot-1** [comment]
@zimengle updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9845/files/5829fc75d7710a31b4b6495423e6323a0448cb40..96db94ba49a6a5ac74b038c758f60a3906232d67)

**Comment by zimengle** [comment]
@nihgwu refine done ，thanks

**Comment by facebook-github-bot-8** [comment]
@zimengle updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9845/files/96db94ba49a6a5ac74b038c758f60a3906232d67..53df7f1c016741116b6de32aefa91156f92270f9)

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -329,6 +329,14 @@ var featured = [
     infoLink: 'https://engineering.huiseoul.com/building-a-conversational-e-commerce-app-in-6-weeks-with-react-native-c35d46637e07',
     infoTitle: 'Building a conversational E-commerce app in 6 weeks with React Native',
   },
+  {
+    name: 'Baidu(手机百度)',
+    icon: 'https://ss0.bdstatic.com/-0U0bnSm1A5BphGlnYG/tam-ogel/8b8eb88700776fe293f4f9a74a73aa18_121_121.jpg',
+    linkPlayStore: 'http://shouji.baidu.com/software/9896302.html',
+    linkAppStore: 'https://itunes.apple.com/en/app/shou-ji-bai-du-hai-liang-xin/id382201985?l=en&mt=8',
+    infoLink: 'http://baike.baidu.com/link?url=TW8YhcVN4tO_Jz5VqMclCjGhf12EEqMD_TeVC6efe2REZlx80r6T0dX96hdmNl36XogLyExXzrvFU9rFeqxg_K',
+    infoTitle: 'Baidu is a Search Engine APP that has been used by 600 millions users.',
```
Comment (lacker) [inline website/src/react-native/showcase.js]: Just a grammatical nitpick: could you rephrase this sentence as:

"Baidu is a search engine that has 600 million users."

**Comment by facebook-github-bot-0** [comment]
@zimengle updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9845/files/53df7f1c016741116b6de32aefa91156f92270f9..1a5a1fcccfcfd93c17eac8cac2118c4871bf3acb)

File: website/src/react-native/showcase.js (hunk 1 above)
Comment (zimengle) [inline website/src/react-native/showcase.js]: @lacker done..thanks for your advice.

**Comment by lacker** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-4** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3854942) to review internal test results.
